packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -22,65 +23,29 @@ const { prefix } = settings;
 /**
  * LightboxMediaViewer Component
  *
- * @param {object} props props object
- * @param {string} props.title LightboxMediaViewer media title
- * @param {string} props.copy LightboxMediaViewer media short description
- * @param {string} props.image LightboxMediaViewer responsive image object
- * @param {boolean} props.open sets whether the modal is open/close
- * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @param {object} modalProps props object
+ * @param {string} modalProps.title LightboxMediaViewer media title
+ * @param {string} modalProps.copy LightboxMediaViewer media short description
+ * @param {string} modalProps.image LightboxMediaViewer responsive image object
+ * @param {boolean} modalProps.open sets whether the modal is open/close
  * @returns {*} JSX Object
  */
-const LightboxMediaViewer = ({ title, copy, image, open, onClose }) => {
-  /**
-   * Set default image
-   *
-   * @param {*} image responsive image object
-   * @returns {string} default image url
-   */
-  const defaultImage = image => {
-    return image.uri.md
-      ? image.uri.md
-      : image.uri.lg
-      ? image.uri.lg
-      : image.uri.sm;
-  };
-
-  if (!image.uri.lg) {
-    return null;
-  }
+const LightboxMediaViewer = ({ title, images, copy, ...modalProps }) => {
```

Comment:
Following that comment this needs to be `image`.

Reply:
I thought from the PR review comments it could be array of images so that is the reason I have changed to `images`. 
![image](https://user-images.githubusercontent.com/53789187/72638208-f6cc0280-3930-11ea-9790-133e6a0f0083.png)

Reply:
It still is an array, but it's an array of sources inside of the Image data model. Take a look at what Karthik did. The image component has been better defined since this comment was made, and it was never intended to list out multiple images at once, but allow us to leverage multiple breakpoints.

https://ibm.box.com/s/1bbrspqkipecv2o010l8czzese084cix

Reply:
Okay.